cruise-control/src/yaml/endpoints/fixOfflineReplicas.yaml

```diff
@@ -0,0 +1,150 @@
+FixOfflineReplicasEndpoint:
+  post:
+    operationId: fixOfflineReplicas
+    summary: Fix offline replicas by moving currenlty offline replicas to healthy brokers
+    parameters:
+      - name: dryrun
+        in: query
+        description: Whether dry-run the request or not
+        schema:
+          type: boolean
+          default: true
+      - name: goals
```

Comment:
Currently we do not check for `optional` for endpoint parameters since we do not have metadata in Request Java class to infer whether a parameter is required or not. 
I prefer solve this in the in the following auto-gen patch.